src/shared/components/pipeline-run-logs/StatusIcon.tsx

```diff
@@ -66,17 +66,26 @@ export const StatusIconWithText: React.FC<StatusIconProps & { text?: string }> =
   ...others
 }) => {
   return (
-    <Badge isRead>
+    <Label>
```

Comment:
Label here is missing color property, so the background is still grey.

![image](https://user-images.githubusercontent.com/9964343/193259511-3e2423d8-2a2f-4cea-9e05-c9b6e890d29a.png)